src/components/BillContext/BillContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useState } from 'react';
+
+export const BillContext = createContext();
+
+export const BillProvider = ({ children }) => {
+  const [bill, setBill] = useState(0);
+  const [person, setPerson] = useState(0);
```

Comment:
Rename this state variable, person should not be assigned to 0, if you're going for something like noOfPersons, that's fine, but your variable names should be self-documenting.